apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -270,13 +275,16 @@ async function reset({ client }: Context): Promise<void> {
 async function scan({ client, workspace }: Context): Promise<SheetOf<string>> {
   assert(client);
   debug('Scanning');
+  const isUltrasonicDisabled = workspace.store.getIsUltrasonicDisabled();
   const scanResult = await client.scan({
     wantedScanSide: ScanSide.A_AND_B,
     resolution: ImageResolution.RESOLUTION_200_DPI,
     imageColorDepth: ImageColorDepthType.Grey8bpp,
     formStandingAfterScan: FormStanding.HOLD_TICKET,
     // The A4 scanner does not support double sheet detection.
-    doubleSheetDetection: DoubleSheetDetectOpt.DetectOff,
+    doubleSheetDetection: isUltrasonicDisabled
```

Comment:
Given that the custom lib calls this "double sheet detection," it seems a bit weird that we then start calling it "ultrasonic" at higher levels in the stack. For consistency, I think it would be clearer to call it "double sheet detection" throughout the system. I recognize it's kind of an annoying change to make but maybe is worth it if you agree?

Reply:
yeah good point. Going to merge this as is but I made a note to make a follow up PR for this